Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -68,23 +74,38 @@ struct PitchSpeller {
         self.pitches = pitches
         self.parsimonyPivot = parsimonyPivot
         self.pitchNodes = PitchSpeller.internalNodes(pitches: pitches)
-        self.flowNetwork = FlowNetwork(source: -2, sink: -1, internalNodes: pitchNodes)
+        self.flowNetwork = FlowNetwork(
+            source: Cross<Int, Tendency>(-1, .down),
+            sink: Cross<Int, Tendency>(-1, .up),
+            internalNodes: pitchNodes
+        )
     }
 
     /// - Returns: An array of `SpelledPitch` values in the order in which the original
     /// unspelled `Pitch` values are given.
     func spell() -> [SpelledPitch<EDO12>] {
 
         var assignedNodes: [AssignedNode] {
-            let (sourceSide, sinkSide) = flowNetwork.minimumCut
+            var (sourceSide, sinkSide) = flowNetwork.minimumCut
+            sourceSide.remove(flowNetwork.source)
+            sinkSide.remove(flowNetwork.sink)
             let downNodes = sourceSide.map { index in AssignedNode(index, .down) }
             let upNodes = sinkSide.map { index in AssignedNode(index, .up) }
             return downNodes + upNodes
         }
 
         return assignedNodes
-            .sorted { $0.index < $1.index }
-            .dropFirst(2)
+            .sorted {
```

Comment:
Would this be semantically equivalent?

```Swift
.sorted {
    if $0.index.a < $1.index.a { return true }
    return $0.index.a == $1.index.a && UInt8($0.index.b.rawValue) < UInt8($1.index.b.rawValue)
}
```

Reply:
Overall, my intuition is that this comparison should be in the conformance of `AssignedNode: Comparable`. I'm gonna pull down to poke at it.

Reply:
See #125.